examples/manual/manual.md

```diff
@@ -0,0 +1,54 @@
+# End to end TMT tutorial
+## Setup
+1. Install tmt
+```bash
+sudo dnf install -y tmt-all
+```
+2. Create and switch to a brand new git repo
+```bash
+cd `mktemp -d`
+git init
+```
+## Test steps
+### Step 1
```

Comment:
What about using the level three heading `###` for a human-readable name of the step and level four `#### Step` for the step description? Here it could look like this:

```markdown
### Usage Message
#### Step
tmt --help
#### Result
Text similar to the one bellow is displayed...
```

Reply:
OK, that will leave level three available for user.